logging/bin/deploy_fluentbit_opensearch.sh

```diff
@@ -150,6 +177,7 @@ helm $helmDebug upgrade --install --namespace $LOG_NS v4m-fb  \
   --version $FLUENTBIT_HELM_CHART_VERSION \
   --values logging/fb/fluent-bit_helm_values_opensearch.yaml  \
   --values $openshiftValuesFile \
+  --values $airgapValuesFile \
```

Comment:
Don't we need/want to reference $AIRGAP_HELM_REPO below (on line 182)?   Might be good to split line 182 below so the Helm chart name sits on its own line.

Reply:
Actually remembered to include the airgapValuesFile in this push.

Reply:
wow!  I should have caught that...good catch!